python/tests/compat/test_mantid.py

```diff
@@ -466,6 +466,18 @@ def test_to_quat_from_vectors(self):
         self.assertTrue(
             np.allclose(a, q.to_rotation_matrix().dot(b), rtol=0.0, atol=1e-9))
 
+    def test_validate_units(self):
+        acceptable = ["wavelength", sc.Dim.Wavelength]
```

Comment:
`sc.Dim` is deprecated and should be removed soon. Is there a way to write this without referencing it here?

Reply:
In this case it came about from

```
unit = wavelength["sample"].dims[0]
sc.compat.mantid.to_workspace_2d(..., unit)
```

I agree with the fact it's deprecated, however since `sc.Dim` still exists this is still a bug until the work to remove it is completed
I did think to myself should I just go ahead and remove `Dim.`, but it falls outside of the scope of what I'm currently doing